packages/terra-image/src/Image.jsx

```diff
@@ -23,6 +23,10 @@ const propTypes = {
    * The text content that specifies an alternative text for an image.
    */
   alt: PropTypes.string.isRequired,
+  /**
+   * The source string for the image which will be displayed during loading and in case of src load failure.
+   */
+  placeholderSrc: PropTypes.string,
```

Comment:
Could we change this to a node for greater flexibility?

Reply:
updated